### PR TITLE
fix(prisma-client-reference): Add missing raw methods

### PR DESCRIPTION
**Pull request by janpio**
Let's at least mention them with a link to where they are currently documented.

### CODE REVIEW

File: content/200-orm/500-reference/050-prisma-client-reference.mdx [hunk 1]
```diff
@@ -1639,6 +1639,16 @@ const groupUsers = await prisma.user.groupBy({
...
 
+### `findRaw()`
+
+See: [Raw database access (`findRaw()`)](/orm/prisma-client/queries/raw-database-access/raw-queries#findRaw).
+
+### `aggreagateRaw()`
+
+See: [Raw database access (`aggreagateRaw()`)](/orm/prisma-client/queries/raw-database-access/raw-queries#aggreagateRaw).
+
+
```
Comment (janpio) [inline content/200-orm/500-reference/050-prisma-client-reference.mdx]:
```suggestion
```

File: content/200-orm/500-reference/050-prisma-client-reference.mdx [hunk 2]
```diff
@@ -1639,6 +1639,14 @@ const groupUsers = await prisma.user.groupBy({
 </cmdResult>
 </CodeWithResult>
 
+### `findRaw()`
+
+See: [Raw database access (`findRaw()`)](/orm/prisma-client/queries/raw-database-access/raw-queries#findRaw).
+
+### `aggreagateRaw()`
+
+See: [Raw database access (`aggreagateRaw()`)](/orm/prisma-client/queries/raw-database-access/raw-queries#aggreagateRaw).
```
Comment (janpio) [inline content/200-orm/500-reference/050-prisma-client-reference.mdx]:
```suggestion
See: [Raw database access (`findRaw()`)](/orm/prisma-client/queries/raw-database-access/raw-queries#findraw).

### `aggreagateRaw()`

See: [Raw database access (`aggreagateRaw()`)](/orm/prisma-client/queries/raw-database-access/raw-queries#aggreagateraw).
```

File: content/200-orm/500-reference/050-prisma-client-reference.mdx [hunk 3]
```diff
@@ -1639,6 +1639,14 @@ const groupUsers = await prisma.user.groupBy({
 </cmdResult>
 </CodeWithResult>
 
+### `findRaw()`
+
+See: [Raw database access (`findRaw()`)](/orm/prisma-client/queries/raw-database-access/raw-queries#findraw).
+
+### `aggreagateRaw()`
+
+See: [Raw database access (`aggreagateRaw()`)](/orm/prisma-client/queries/raw-database-access/raw-queries#aggreagateraw).
```
Comment (janpio) [inline content/200-orm/500-reference/050-prisma-client-reference.mdx]:
```suggestion
See: [Raw database access (`aggreagateRaw()`)](/orm/prisma-client/queries/raw-database-access/raw-queries#aggregateraw).
```

File: content/200-orm/500-reference/050-prisma-client-reference.mdx [hunk 4]
```diff
@@ -5378,13 +5386,21 @@ Example parameter values:
 
 See [middleware examples](/orm/prisma-client/client-extensions/middleware#samples) .
 
-### `$executeRaw()`
+### `$queryRaw`
```
Comment (jharrell) [inline content/200-orm/500-reference/050-prisma-client-reference.mdx:5389]: Does this not have `()` because it's a template string?

**Review by jharrell — APPROVED** [review]

File: content/200-orm/500-reference/050-prisma-client-reference.mdx (hunk 4 above)
Comment (janpio) [reply content/200-orm/500-reference/050-prisma-client-reference.mdx:5389]: Yes, exactly. There is no great way to represent that that I could find :/

**Pull request merged**